webapi/src/main/java/org/sakaiproject/webapi/controllers/TasksController.java

```diff
@@ -286,4 +279,21 @@ private String getTaskAssignedDescription(Long taskId, Site site) {
         return result;
     }
 
+    /**
+     * This method updates a UserTaskAdapterBean object.
+     *
+     * @param bean the UserTaskAdapterBean object to update
+     * @throws IdUnusedException if the specified site ID is invalid
+     */
+    private void updateUserTaskAdapterBean(UserTaskAdapterBean bean) throws IdUnusedException {
+        Site site = siteService.getSite(bean.getSiteId());
+
+        bean.setSiteTitle(site.getTitle());
+        if (StringUtils.isNotBlank(bean.getReference()) && !bean.getReference().startsWith("/user/")) {
+            entityManager.getUrl(bean.getReference(), Entity.UrlType.PORTAL).ifPresent(u -> bean.setUrl(u));
+        }
+        bean.setTaskAssignedTo(getTaskAssignedDescription(bean.getTaskId(), site));
+    }
+
+
```

Comment:
You don't need these blank lines at the bottom like this.

Reply:
Sure! Thanks for the suggestion!